template/template_example.Rmd

```diff
@@ -11,7 +11,7 @@ output:
 
 # Purpose of the analysis:
 
-<<WHAT DOES THIS ANALSIS DO AND WHEN WOULD YOU USE IT?>>
+<!--WHAT DOES THIS ANALYSIS DO AND WHEN WOULD YOU USE IT?-->
```

Comment:
So the idea here would be to add context based on this prompt but also leave this prompt in the notebook above the context once added?

Reply:
Yes. That's the idea. That way when we review things we know where to expect added content.

Reply:
Gotcha 👍